python/metatensor-learn/metatensor/learn/nn/layer_norm.py

```diff
@@ -128,44 +129,50 @@ class InvariantLayerNorm(Module):
     Refer to the :py:class`torch.nn.LayerNorm` documentation for a more detailed
     description of the other parameters.
 
-    Each parameter can be passed as a single value of its expected type, which is used
-    as the parameter for all blocks. Alternatively, they can be passed as a list to
-    control the parameters applied to each block indexed by the invariant keys in :param
-    in_keys: at numeric poositions :param invariant_key_idxs:.
+    Each parameter is passed as a single value of its expected type, which is used
+    as the parameter for all blocks.
 
     :param in_keys: :py:class:`Labels`, the keys that are assumed to be in the input
         tensor map in the :py:meth:`forward` method.
-    :param invariant_key_idxs: list of int, the indices of the invariant keys present in
-        `in_keys` in the input :py:class:`TensorMap`. Only blocks for these keys will
-        have layer norm applied. The other blocks will have the identity operator
-        applied.
     :param in_features: list of int, the number of features in the input tensor for each
         block indexed by the keys in :param in_keys:. If passed as a single value, the
         same number of features is assumed for all blocks.
     :param out_properties: list of :py:class`Labels` (optional), the properties labels
         of the output. By default (if none) the output properties are relabeled using
         Labels.range.
+    :param invariant_keys: a :py:class:`Labels` object that is used to select the
+        invariant keys from ``in_keys``. If not provided, the invariant keys are assumed
+        to be those where key dimensions ``["o3_lambda", "o3_sigma"]`` are indexed by
```

Comment:
```suggestion
        to be those where key dimensions ``["o3_lambda", "o3_sigma"]`` are equal to
```

and same everywhere!